source/clear-linux/get-started/install-configfile.rst

```diff
@@ -0,0 +1,185 @@
+   #. Delete the *post-install* section unless you have post-installation scripts.
+   #. Under *Version*, set a version number. To use the latest version, set to “0”.
+
+   Commonly changed settings are shown in lines 15, 34, 37, and 51 below.
+   See `Installer YAML Syntax`_ for more details.
+
+   .. code-block:: bash
+      :linenos:
+      :emphasize-lines: 14,15,34,37,51
+
+		#clear-linux-config
+
+		# c-basic-offset: 2; tab-width: 2; indent-tabs-mode: nil
+		# vi: set shiftwidth=2 tabstop=2 expandtab:
+		# :indentSize=2:tabSize=2:noTabs=true:
+
+		# File:         developer-live-server.yaml
+		# Use Case:     Live Image which boots into login prompt
+		#               Optionally allows for installing Clear Linux OS
+		#               using the TUI clr-installer by running clr-installer
+
+		# switch between aliases if you want to install to an actual block device
+		# i.e /dev/sda
+		block-devices: [
+		   {name: "bdevice", file: "/dev/sda"}
+		]
+
+		targetMedia:
+		- name: ${bdevice}
+		  type: disk
+		  children:
+		  - name: ${bdevice}1
+		    fstype: vfat
+		    mountpoint: /boot
+		    size: "150M"
+		    type: part
+		  - name: ${bdevice}2
+		    fstype: swap
+		    size: "32M"
+		    type: part
+		  - name: ${bdevice}3
+		    fstype: ext4
+		    mountpoint: /
+		    size: "0"
+		    type: part
+
+		bundles: [os-core, os-core-update, NetworkManager, clr-installer, vim]
+
+		autoUpdate: false
+		postArchive: false
+		postReboot: false
+		telemetry: false
+		iso: true
+		keepImage: true
+		autoUpdate: false
+
+		keyboard: us
+		language: en_US.UTF-8
+		kernel: kernel-native
+
+		version: 30010
+
+.. _install-configfile-yaml-end:
+
+Start the installation with the command:
+
+.. code-block:: bash
+
+   clr-installer --config live-server.yaml
+
+Example 2
+=========
+
+This method uses a saved configuration file from a previous installation,
+which you can use to easily duplicate the installation on additional machines.
+
+Perform the following steps:
+
+#. Open a console window on a system where |CL| was installed to retrieve a
+   copy of the configuration file.
+
+#. In the console window, log in as root and enter your password.
+
+#. Change directory to :file:`/root` and copy the :file:`clr-installer.yaml`
+   file to a thumb drive.
+
+   .. code-block:: bash
+
+   	  cd /root
+   	  cp clr-installer.yaml
+
+Start the installation on the target with the following commands:
+
+#. Go to `Downloads`_ and download the latest Clear Linux OS Server image.
+
+   For example:
+   https://download.clearlinux.org/releases/30010/clear/clear-30010-live-server.iso.xz
+
+#. Follow the instructions to :ref:`bootable-usb` based on your OS.
+
+#. Boot up the USB thumb drive.
+#. Select :guilabel:`Clear Linux OS` from the menu.
+#. In the console window, log in as root and set a password.
+#. Verify you have an IP address (network connection) and configure proxies (if needed).
+#. Start the installation with the command:

```

Comment:
@MCamp859 - pls add a step between lines 169-170 that says "Plug in and mount the USB thumb drive containing the retrieved clr-installer.yaml configuration file."